src/conf.py

```diff
@@ -86,6 +86,9 @@
 intersphinx_mapping = {
     'rose': (
         'http://metomi.github.io/rose/doc/html', None
+    ),
+    'python': (
```

Comment:
The Python reference material defines things like `PYTHONPATH` which is why I've turned it on in inter-sphinx.